### ISSUE TITLE
[Feature request]: Determine printable items in draft printing mode only

### ISSUE THREAD

**Issue by gpsbait** [yrestom/POS-Awesome#268]
- Determine printable items in draft printing mode only
The aim of this is to use the program in restaurants or cafes, to print as to prepare the order
After printing, the customer can order another item, and this item is selected only for printing and not the entire invoice

- Adding a service template such as Delivery, and the template is like Delivery, but it is a percentage of the total before tax

**Comment by gpsbait** [comment]
clarification
In the case of draft printing
That there is a feature to specify all types of items or specific items for printing as a draft

**Comment by yrestom** [comment]
Is that can be done using printing format? if not can you please suggest how it can be done?

**Comment by gpsbait** [comment]
It is not possible via the printing format because this is done on the PRINT DRAFT only and not print upon payment
This is done when you press the Print Draft button, a pop-up window as Held List appears in which all items are present, with a Select All button, or specific items are selected manually for printing

**Comment by yrestom** [comment]
If we add another print format for draft printing in addition to the current one, is that works for you?

**Comment by gpsbait** [comment]
It is not suitable because all items are printed
What do you want to be specific items during printing

- There is another advantage if a specific default printer is specified for the DRAFT  and another print when paying, which will be a great feature

**Comment by yrestom** [comment]
I do not think that it is possible to do it, as now we use the functional process integrated in Frappe to print the invoice as it is, and I do not think that there is a possibility to selectively print the items other than using a custom print format
If you have a way to do that, please share it here
